### PR TITLE
Nat/intellinav

### PR DESCRIPTION
**Pull request by nturinski**
Add intellinav that can be used for PRs in GitHub

### CODE REVIEW

File: .azure-pipelines/windows/intellinav.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+steps:
+- task: vsintellinav.vsck-service-endpoint.build-task.upload-cache-build-task@0
+  displayName: 'VS IntelliNav Upload'
+  inputs:
+    vsckEndpoint: 'azureappservice-dev' # name of the IntelliNav service connection
```
Comment (EricJizbaMSFT) [inline .azure-pipelines/windows/intellinav.yml]: azureappservice?

Comment (nturinski) [reply .azure-pipelines/windows/intellinav.yml]: It was just the generic name I used for the service connection.  I didn't have the foresight to think I'd be doing this on all the extensions 😇

Comment (EricJizbaMSFT) [reply .azure-pipelines/windows/intellinav.yml]: Can we change it?

Comment (nturinski) [reply .azure-pipelines/windows/intellinav.yml]: Yes, I can.  What would you like to name it?

Comment (EricJizbaMSFT) [reply .azure-pipelines/windows/intellinav.yml]: idk something with either azuretools or azcode instead of appservice

Comment (nturinski) [reply .azure-pipelines/windows/intellinav.yml]: `azazurecodetools` it is

**Pull request merged**